doc/src/cylc-user-guide/cug.tex

```diff
@@ -4379,6 +4379,7 @@ \subsubsection{Parameter Expansion}
     [[parameters]]
         obs = ship, buoy, plane
         run = 1..5  # 1, 2, 3, 4, 5
+        idx = 1..9^2  # 1, 3, 5, 7, 9
```

Comment:
I think @matthewrmshin's solution makes sense as it's in line with what you'd expect from something like python's `range(start, end, step=1)` function.